From 3c64f4b806d3106a4b4fda0d4e323dae950407b0 Mon Sep 17 00:00:00 2001
From: Amin Shaikh <ashaikh@google.com>
Date: Thu, 14 Mar 2019 16:33:19 +0000
Subject: [PATCH 43/48] Add bluetooth icon.

Add accent_device_default that uses the light/dark variant based on
light/dark ui mode.

Bug: 126425594
Change-Id: I08640cc161b6d63da42c3f03df7cf41b182b005e
Test: make
---
  core/res/res/drawable/ic_bluetooth.xml       | 25 +++++++++++++++++++
 .../values-night/colors_device_defaults.xml   | 19 ++++++++++++++
 .../res/res/values/colors_device_defaults.xml |  1 +
 3 files changed, 45 insertions(+)
 create mode 100644  core/res/res/drawable/ic_bluetooth.xml
 create mode 100644 core/res/res/values-night/colors_device_defaults.xml

diff --git a/ core/res/res/drawable/ic_bluetooth.xml b/ core/res/res/drawable/ic_bluetooth.xml
new file mode 100644
index 00000000000..74022a2bfac
--- /dev/null
+++ b/ core/res/res/drawable/ic_bluetooth.xml	
@@ -0,0 +1,25 @@
+<!--
+     Copyright (C) 2019 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="@android:color/accent_device_default"
+        android:pathData="M17.71,7.71L12,2h-1v7.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L11,14.41V22h1l5.71-5.71L13.41,12L17.71,7.71z M13,5.83 l1.88,1.88L13,9.59V5.83z M14.88,16.29L13,18.17v-3.76L14.88,16.29z" />
+</vector>
\ No newline at end of file
diff --git a/core/res/res/values-night/colors_device_defaults.xml b/core/res/res/values-night/colors_device_defaults.xml
new file mode 100644
index 00000000000..08ad4926197
--- /dev/null
+++ b/core/res/res/values-night/colors_device_defaults.xml
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2019 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<resources>
+    <color name="accent_device_default">@color/accent_device_default_dark</color>
+</resources>
\ No newline at end of file
diff --git a/core/res/res/values/colors_device_defaults.xml b/core/res/res/values/colors_device_defaults.xml
index 0fe80a154f7..5a963e43183 100644
--- a/core/res/res/values/colors_device_defaults.xml
+++ b/core/res/res/values/colors_device_defaults.xml
@@ -33,6 +33,7 @@
 
     <color name="accent_device_default_light">@color/accent_material_light</color>
     <color name="accent_device_default_dark">@color/accent_material_dark</color>
+    <color name="accent_device_default">@color/accent_device_default_light</color>
 
     <color name="background_device_default_dark">@color/background_material_dark</color>
     <color name="background_device_default_light">@color/background_material_light</color>
-- 
2.17.1

